modules/tinymce/CHANGELOG.md

```diff
@@ -39,7 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `caption`, `address` and `dt` elements no longer incorrectly allow non-inline child elements when the editor schema is set to _HTML 4_. #TINY-9768
 - SVG icons for back and foreground colors now use `class` instead of `id` to identify SVG elements that should change color. #TINY-9844
 - Anchor tag elements — `<a>` — no longer incorrectly allow non-inline child elements when the editor schema is set to _HTML 4_. #TINY-9805
-- Help dialog was restored to `medium` width for better readability. #TINY-9947
+- Help dialog was restored to `medium` width for better readability #TINY-9947
```

Comment:
```suggestion
- Help dialog was restored to `medium` width for better readability. #TINY-9947
```

It will undo the change, which should not be in here